vpn/vpn.go

```diff
@@ -17,19 +17,28 @@ import (
 	"github.com/sagernet/sing-box/experimental/cachefile"
 	"github.com/sagernet/sing-box/experimental/libbox"
 
+	"github.com/getlantern/radiance"
 	"github.com/getlantern/radiance/common"
 	"github.com/getlantern/radiance/servers"
 )
 
 // QuickConnect automatically connects to the best available server in the specified group. Valid
 // groups are [ServerGroupLantern], [ServerGroupUser], "all", or the empty string. Using "all" or
 // the empty string will connect to the best available server across all groups.
-func QuickConnect(group string, platIfce libbox.PlatformInterface) error {
+func QuickConnect(group string, platIfce libbox.PlatformInterface, options radiance.Options) error {
```

Comment:
We should avoid import `radiance` to ensure that they remain completely independent. 
```suggestion
func QuickConnect(group string, platIfce libbox.PlatformInterface, dataDir, logDir, logLevel string) error {
```